webv2/web.go

```diff
@@ -2304,6 +2304,8 @@ func getGeneratedResources(w http.ResponseWriter, r *http.Request) {
 	generatedResources.BucketUrl = fmt.Sprintf("https://console.cloud.google.com/storage/browser/%v", sessionState.Bucket+sessionState.RootPath)
 	generatedResources.ShardToDataflowMap = make(map[string]ResourceDetails)
 	generatedResources.ShardToDatastreamMap = make(map[string]ResourceDetails)
+	generatedResources.ShardToPubsubTopicMap = make(map[string]ResourceDetails)
+	generatedResources.ShardToPubsubSubscriptionMap = make(map[string]ResourceDetails)
```

Comment:
This map is used in angular code to iterate over the ResouceDetails and print the details. Changing the Value type of this object from ResourceDetails to Generic object would require a lot of changes and also it would be difficult to iterate and read in angular code.